<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DiaGraphica</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Graph</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Number of states:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>102</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="numberOfStates">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Number of transitions:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>102</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="numberOfTransitions">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Attributes</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="attributes">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>false</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::InternalMove</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>#</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Type</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Cardinality</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Domain</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="domain">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::InternalMove</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <column>
            <property name="text">
             <string>#</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Value</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string># Nodes</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>% Nodes</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="examinerWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QStackedWidget" name="mainViewStack">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_3">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSplitter" name="splitter_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QWidget" name="arcDiagramWidget" native="true"/>
           <widget class="QStackedWidget" name="analysisStack">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="simulatorWidget"/>
            <widget class="QWidget" name="traceWidget"/>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_4">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QWidget" name="diagramEditorWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QToolButton" name="selectTool">
             <property name="text">
              <string>Select</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/images/selecticon.png</normaloff>:/images/selecticon.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string>toolButtonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="noteTool">
             <property name="text">
              <string>Note</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/images/note.png</normaloff>:/images/note.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string>toolButtonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="configureTool">
             <property name="text">
              <string>Configure</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/images/dof.png</normaloff>:/images/dof.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string>toolButtonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="rectangleTool">
             <property name="text">
              <string>Rectangle</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/images/rectangle.png</normaloff>:/images/rectangle.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string>toolButtonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="ellipseTool">
             <property name="text">
              <string>Ellipse</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/images/ellipse.png</normaloff>:/images/ellipse.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string>toolButtonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="lineTool">
             <property name="text">
              <string>Line</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/images/line.png</normaloff>:/images/line.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string>toolButtonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="arrowTool">
             <property name="text">
              <string>Arrow</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/images/arrow.png</normaloff>:/images/arrow.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string>toolButtonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="doubleArrowTool">
             <property name="text">
              <string>Double Arrow</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/images/darrow.png</normaloff>:/images/darrow.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string>toolButtonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="colorTool">
             <property name="text">
              <string>Color</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/images/fillcol.png</normaloff>:/images/fillcol.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="lineColorTool">
             <property name="text">
              <string>Line Color</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/images/linecol.png</normaloff>:/images/linecol.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="showGridButton">
             <property name="text">
              <string>Show Grid</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/images/showgrid.png</normaloff>:/images/showgrid.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="snapToGridButton">
             <property name="text">
              <string>Snap to Grid</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/images/snapgrid.png</normaloff>:/images/snapgrid.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>873</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionLoadAttributes"/>
    <addaction name="actionSaveAttributes"/>
    <addaction name="separator"/>
    <addaction name="actionOpenDiagram"/>
    <addaction name="actionSaveDiagram"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>&amp;Mode</string>
    </property>
    <addaction name="actionAnalysisMode"/>
    <addaction name="actionTraceMode"/>
    <addaction name="actionEditMode"/>
   </widget>
   <widget class="QMenu" name="menuAttributes">
    <property name="title">
     <string>&amp;Attributes</string>
    </property>
    <addaction name="actionClusterNodes"/>
    <addaction name="actionViewTrace"/>
    <addaction name="separator"/>
    <addaction name="actionDistributionPlot"/>
    <addaction name="actionCorrelationPlot"/>
    <addaction name="actionCombinationPlot"/>
    <addaction name="separator"/>
    <addaction name="actionDuplicate"/>
    <addaction name="actionRenameAttribute"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuDomain">
    <property name="title">
     <string>&amp;Domain</string>
    </property>
    <addaction name="actionGroup"/>
    <addaction name="actionUngroup"/>
    <addaction name="actionRenameValue"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionSettingsGeneral"/>
    <addaction name="actionSettingsArcDiagram"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMode"/>
   <addaction name="menuAttributes"/>
   <addaction name="menuDomain"/>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="actionLoadAttributes">
   <property name="text">
    <string>Open Attributes...</string>
   </property>
  </action>
  <action name="actionSaveAttributes">
   <property name="text">
    <string>Save Attributes...</string>
   </property>
  </action>
  <action name="actionOpenDiagram">
   <property name="text">
    <string>Open Diagram...</string>
   </property>
  </action>
  <action name="actionSaveDiagram">
   <property name="text">
    <string>Save Diagram...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionAnalysisMode">
   <property name="text">
    <string>&amp;Simulation Mode</string>
   </property>
  </action>
  <action name="actionEditMode">
   <property name="text">
    <string>&amp;Edit Mode</string>
   </property>
  </action>
  <action name="actionTraceMode">
   <property name="text">
    <string>&amp;Trace Mode</string>
   </property>
  </action>
  <action name="actionClusterNodes">
   <property name="text">
    <string>Cluster Nodes</string>
   </property>
  </action>
  <action name="actionViewTrace">
   <property name="text">
    <string>View Trace</string>
   </property>
  </action>
  <action name="actionDistributionPlot">
   <property name="text">
    <string>Distribution Plot</string>
   </property>
  </action>
  <action name="actionCorrelationPlot">
   <property name="text">
    <string>Correlation Plot</string>
   </property>
  </action>
  <action name="actionCombinationPlot">
   <property name="text">
    <string>Combination Plot</string>
   </property>
  </action>
  <action name="actionDuplicate">
   <property name="text">
    <string>Duplicate</string>
   </property>
  </action>
  <action name="actionRenameAttribute">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionGroup">
   <property name="text">
    <string>Group</string>
   </property>
  </action>
  <action name="actionUngroup">
   <property name="text">
    <string>Ungroup</string>
   </property>
  </action>
  <action name="actionRenameValue">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="actionSettingsGeneral">
   <property name="text">
    <string>General</string>
   </property>
  </action>
  <action name="actionSettingsArcDiagram">
   <property name="text">
    <string>Arc Diagram</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="toolButtonGroup"/>
 </buttongroups>
</ui>
